l10n/en-US/viewer.ftl

```diff
@@ -348,9 +348,9 @@ pdfjs-editor-free-highlight-thickness-input = Thickness
 pdfjs-editor-free-highlight-thickness-title =
     .title = Change thickness when highlighting items other than text
 
-pdfjs-free-text =
+pdfjs-free-text2 =
```

Comment:
```suggestion
pdfjs-empty-text =
```

I wonder why `free` was chosen in the first place for this ID? I assume this is the default text when the editor is empty.

Reply:
Because this localization is for the FreeText editor, and the aria-label applies even when after the user has entered a string. Hence having "empty" in the name seems strange/wrong to me.

Reply:
Sounds good, feel free to ignore the suggestion.